### PR TITLE
first refactorization after split. Only taking into account scipion-p…

### PR DESCRIPTION
**Pull request by pconesa**
…yworkflow.

### CODE REVIEW

File: scipion [hunk 1]
```diff
@@ -32,6 +32,44 @@ import sys
 import os
 from os.path import join, exists, dirname, expanduser
 import subprocess
+import pyworkflow
```
Comment (delarosatrevin) [inline scipion]: If 'scipion' script will be installing python environment,  probably this import will not work.
Nonetheless, probably is a good idea to separte the install logic to another script (e.g scipion-install.py or scipion-build.py) and assume here that Python environment is ready and Scipion configuration correct.

File: scipion [hunk 2]
```diff
@@ -328,19 +331,20 @@ def main():
...
-        for d, path in DIRS_GLOBAL.items():
-            if not exists(path):
-                sys.stderr.write('Missing %s folder: %s\n' % (d, path))
-                ok = False
-#         if not exists(join(VARS['SCIPION_SOFTWARE'], 'log', 'success.log')):
-#             ok = False
-        if not ok:
-            sys.exit("There is a problem with the installation. Please run:\n"
-                     "  %s install" % sys.argv[0])
+## Pconesa Commented: we might not need this or do it differently.
```
Comment (delarosatrevin) [inline scipion]: I agree, configuration could be done separately.

File: scripts/tutorial.py [hunk 3]
```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+# **************************************************************************
+# *
+# * Authors:     J.M. De la Rosa Trevin (jmdelarosa@cnb.csic.es)
```
Comment (delarosatrevin) [inline scripts/tutorial.py]: Please update my new affiliation.

File: scripts/tutorial.py [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+# *
+# * Unidad de  Bioinformatica of Centro Nacional de Biotecnologia , CSIC
+# *
+# * This program is free software; you can redistribute it and/or modify
+# * it under the terms of the GNU General Public License as published by
+# * the Free Software Foundation; either version 2 of the License, or
```
Comment (delarosatrevin) [inline scripts/tutorial.py]: I started updating to GPLv3...although not sure about the differences.

Comment (azazellochg) [reply scripts/tutorial.py]: We need agreement of all authors to officially update to gpl3 :)

Comment (azazellochg) [reply scripts/tutorial.py]: The main difference is that gpl3 forbids tivoization

Comment (delarosatrevin) [reply scripts/tutorial.py]: Uppss...my fault that I was changing it so happily :(
Are you OK with gplv3? Btw, what is tivoization?

Comment (delarosatrevin) [reply scripts/tutorial.py]: I just googled about it.

Comment (delarosatrevin) [reply scripts/tutorial.py]: I don't think it might be an issue for this project.

**Comment by azazellochg** [comment]
Sure, I'm both hands in for version 3!

**Comment by delarosatrevin** [comment]
Great! @pconesa and @fonsecareyna82 will check with the rest of the people in Madrid.

File: scipion (hunk 1 above)
Comment (pconesa) [reply scipion]: Yep, I've been assuming for now that we have already a python (clean environment from conda or python venv) and later we can focus on what to do if we need to install python.

File: scipion/install/script.py [hunk 5]
```diff
@@ -295,60 +295,61 @@ def removeByPattern(pattern):
...
-    env.addPipModule('jinja2', '2.7.3', default=False)
-    env.addPipModule('tornado', '4.0.2', default=False)
-    env.addPipModule('ipython', '2.1.0', target='IPython', default=False)
-    cython = env.addPipModule('cython', '0.22', target='Cython-0.22*', default=False)
-    cythongsl = env.addPipModule('cythongsl', '0.2.1',
-                                 target='CythonGSL-0.2.1*',
-                                 default=False, deps=[cython])
-    env.addPipModule('scikit-learn', '0.17', target='scikit_learn*',
-                     default=False, deps=[cython])
+    # All pip modules can no be defined in it's correspondent requirements.txt
```
Comment (delarosatrevin) [inline scipion/install/script.py]: no -> now?

**Pull request merged**